rllib/utils/policy.py

```diff
@@ -83,6 +83,11 @@ def create_policy_for_framework(
         session_creator: An optional tf1.Session creation callable.
         seed: Optional random seed.
     """
+    from ray.rllib.algorithms.algorithm_config import AlgorithmConfig
```

Comment:
Making sure all Policies are still created using old-style config dicts, NOT AlgorithmConfig objects.